src/main/resources/io/jenkins/plugins/analysis/core/steps/RecordIssuesStep/help-buildOnError.html

```diff
@@ -0,0 +1,2 @@
+By default, static analysis results are only recorded for stable or unstable builds, but not for failed builds:
+analysis results might be inaccurate if the build failed. If you want the build to fail on error kindly tick this checkbox.
```

Comment:
If there are errors while scanning the log or files for issues (e.g., file pattern matches no files, source files could not be copied, etc.) then the warning plugin will show these errors in a separate view but does not alter the build state. If you would rather like to fail the build on such errors, please tick this checkbox.